src/typeorm-aurora-data-api-driver.ts

```diff
@@ -1,95 +1,113 @@
 // @ts-ignore
 import createDataApiClient from 'data-api-client'
 
-const quoteCharacters = ['\'', '"']
-
-export default class DataApiDriver {
-  public static transformQueryAndParameters(query: string, parameters?: any[]): any {
-    let numberOfParametersInQueryString = 0
-
-    let newQueryString = ''
-    let currentQuote = null
-
-    for (let i = 0; i < query.length; i += 1) {
-      const currentCharacter = query[i]
-      const currentCharacterEscaped = i !== 0 && query[i - 1] === '\\'
-
-      if (currentCharacter === '?' && !currentQuote) {
+const transformQuery = (query: string, parameters?: any[]): [string, number] => {
+  const quoteCharacters = ['\'', '"']
+
+  let newQueryString = ''
+  let currentQuote = null
+  let numberOfParametersInQueryString = 0
+  let extendedParameterCount = 0
```

Comment:
Not sure we need the extended word.

Reply:
I call this extendedParameterCount in order to be in parity with the transformQuery function. I think this makes sense?